guides/simple/debounce-guide.md

```diff
@@ -0,0 +1,36 @@
+---
+description:
+layout:
+  title:
+    visible: true
+  description:
+    visible: false
+  tableOfContents:
+    visible: true
+  outline:
+    visible: true
+  pagination:
+    visible: true
+---
+
+# Making a debounce script
+
+## Introduction
+
+You make a simple script that when you touch, it gives you a tool!
+
+You test it out, but the script rapidly and spontaneously giving tools. That's because you don't have a debounce part of your script.
+
+### How to make it
+
+To make a debounce script, we need to create a variable. This variable will track when the tool has been touched and on cooldown or not. Then, before the event block, you want to set the variable to false,
+to first, indicate that the part hasn't been touched when the game loads, and second, to allow the following if statement to work. Just after the event block, add an if statement. Now, we're going to set the first
+parameter of the if statement to be the variable we just created. The second parameter will be set to false. (Click the dropdown menu, select "Bool" then type in the name of your variable) Next, set the variable back to  After, insert your code for whatever you want to do! The next step is to add the actual delay, and to set the variable back to false.
+
+<img width="327" alt="image" src="https://cdn.discordapp.com/attachments/1127345394779689082/1128129802776363018/image.png">
```

Comment:
Uploading the images to github is a better option instead of relying on discord's CDN. Please change the file accordingly.

Reply:
How should I do that?

Reply:
1.  Drag an image into the comment field.
2. Wait for the upload process to finish.
3. Copy the URL and use it in your Markdown file.

Reply:
I have made the improvements you mentioned.